.env

```
@@ -0,0 +1,4 @@
+PORT = 3001
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)